frontend/src/modules/experimentRuns/actions.js

```diff
@@ -119,7 +119,39 @@ export const listTemplateActions = () => (dispatch, getState) => {
     return dispatch(networkAction(LIST_TEMPLATE_ACTIONS, api.experimentRuns.template.actions()));
 };
 
+export const listProcessProperties = (id) => async (dispatch, getState) => {
+    if (getState().propertyValues.isFetching)
+        return;
 
+    const { itemsByID, propertyValuesByID } = getState().processes    
```

Comment:
propertyValuesByID is not part of the redux state and the var is always undefined. I think you want `propertyValues.itemsByID`.

Reply:
I think I resolved it https://github.com/c3g/freezeman/pull/305/commits/ce584414740e3b3fdc0e7aa87a8f2fef2ac3ffdf?